src/Illuminate/Database/Schema/BlueprintState.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace Illuminate\Database\Schema;
+
+use Illuminate\Database\Connection;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\Grammars\Grammar;
+use Illuminate\Support\Fluent;
+
+class BlueprintState
+{
+    /**
+     * The blueprint instance.
+     *
+     * @var \Illuminate\Database\Schema\Blueprint
+     */
+    protected $blueprint;
+
+    /**
+     * The connection instance.
+     *
+     * @var \Illuminate\Database\Connection
+     */
+    protected $connection;
+
+    /**
+     * The grammar instance.
+     *
+     * @var \Illuminate\Database\Schema\Grammars\Grammar
+     */
+    protected $grammar;
+
+    /**
+     * The columns.
+     *
+     * @var \Illuminate\Database\Schema\ColumnDefinition[]
+     */
+    private $columns = [];
```

Comment:
You may remove redundant initializers (`= [];`) like below since they are assigned immediately in the constructor. 

`private $columns;`

Reply:
Thanks!